src/main/scala/cromwell/util/TryUtil.scala

```diff
@@ -7,6 +7,12 @@ import com.typesafe.scalalogging.LazyLogging
 import scala.concurrent.duration.Duration
 import scala.util.{Success, Failure, Try}
 
+case class AggregatedException[A](exceptions: Seq[Failure[A]]) extends Exception {
+  override def getMessage: String = {
+    exceptions.map(_.exception.getMessage).mkString("\n")
+  }
+}
```

Comment:
yeah this does have the air of `Validation` about it but I guess that's not really worth biting off right now